### ISSUE TITLE
TrackPlayerEvents needs to be added to typescript defs

### ISSUE THREAD

**Issue by rpavlovs** [react-native-kit/react-native-track-player#919]
**Describe the bug**
Importing TrackPlayerEvents produces a TypeScript error.

**To Reproduce/Code**
```js
import { TrackPlayerEvents } from 'react-native-track-player'
// => Module ".../node_modules/react-native-track-player" has no exported member 'TrackPlayerEvents'.ts(2305)
```

**Environment**
`react-native-track-player: 1.2.3`

**Comment by rpavlovs** [comment]
Yay, thanks! :)

**Comment by shivamjjha** [comment]
I was still getting this issue, and it resolved by doing [this](https://github.com/react-native-kit/react-native-track-player/issues/1026#issuecomment-759076563) in my `Typescript` file:

```
import {
  useTrackPlayerProgress,
  useTrackPlayerEvents,
} from "react-native-track-player/lib/hooks"
```

Also, I need to create a file of extension `.d.ts` and put this line:
`declare module 'react-native-track-player/lib/hooks';`

**Comment by curiousdustin** [comment]
Released in [1.2.4](https://github.com/react-native-kit/react-native-track-player/releases/tag/v1.2.4)
